external/abi/set/TradeModule.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I assume that this is our own deployment of `TradeModule.json`. In that case maybe we should put it in `external/abi/index-protocol` or something like that to keep abis of the original and fork deployments separate.